CRM/Contact/Page/View/UserDashBoard.php

```diff
@@ -214,7 +214,6 @@ public function run() {
    * @return array
    *   (reference) of action links
    */
-  static
```

Comment:
A quick grep for "UserDashBoard" turns up this line:

```php
CRM/Contact/BAO/Relationship.php:      $links = CRM_Contact_Page_View_UserDashBoard::links();
```

which suggests that the function should be static.